### PR TITLE
Enable testing on ppc64le architecture

### LINKED ISSUE #135: Enable opentracing-cpp on ppc64le arch #94
Hey!

Thanks for the awesome readme/tutorials on opentracing-cpp. I could build opentracing-cpp from source & run the tests outlined successfully on ppc64le.

Incase, if you are wondering what ppc64le architecture is?
It is a RISC Architecture and is IBM's prodigy, IBM recently has made it's ISA (Instruction Set Architecture) opensource,
doing so they have contributed back to opensource world. Many of the pioneers of banking, HPC industries today run on ppc64le architecture.

As an ongoing effort to enable opensource projects to which ppc64le architecture can add value to, we are trying to enable this project. We have seen absolute no deviation in build/test instructions. I will soon submit a PR to enable build/testing on ppc64l architecture as well.

### PR DESCRIPTION
**Pull request by krishvoor**
This PR Fixes #135 

This PR enables opentracing-cpp on ppc64le architecture
and CI/CD build and tests ( if any ) are triggered via travis.yml

### CODE REVIEW

File: ci/setup_linux_environment.sh [hunk 1]
```diff
@@ -2,14 +2,27 @@
...
-sudo apt-get install software-properties-common
+-o Dir::Etc::sourceparts="-" \
+-o APT::Get::List-Cleanup="0"
+
+fi
 
-sudo add-apt-repository ppa:george-edison55/cmake-3.x -y
 sudo apt-get update
-sudo apt-get install cmake
+sudo apt-get install openjdk-8-jdk bazel -y
```
Comment (yurishkuro) [inline ci/setup_linux_environment.sh:26]: Why do we need a JDK for compiling C++ project? Why do we need bazel?

the build seems to be failing trying to download or install cassandra, why is it needed?

Comment (krishvoor) [reply ci/setup_linux_environment.sh:26]: Hi,

I was wondering the same, but when I tried looking into this PR (https://github.com/opentracing/opentracing-cpp/pull/90/) it looks like a bazel CI was being targeted to be setup.

I will be more than happy to resubmit and realign the build/test process per the documentation.

Comment (yurishkuro) [reply ci/setup_linux_environment.sh:26]: Ah, I didn't realize this repo uses bazel, and I guess that explains the JDK, but not Cassandra.

Comment (krishvoor) [reply ci/setup_linux_environment.sh:26]: I still cannot figure out how - Cassandra came into the picture of nowhere, 
but I guess this is not getting introduced by this PR.

Let me know if I should bump up the ubuntu version from trusty to xenial and give it a try, to see if the issue persists?

**Comment by yurishkuro** [comment]
I restarted the build, it still failed so it's probably not a transient problem.

**Comment by krishvoor** [comment]
> I restarted the build, it still failed so it's probably not a transient problem.

Agreed, I bumped the Ubuntu Release to Xenial - these jobs go through, I am working on ppc64le build
and will hopefully fix it.

**Comment by krishvoor** [comment]
@yurishkuro the PR fixes most of the issues that were uncovered during this exercise.
Would request to help review & merge as required.

**Comment by yurishkuro** [comment]
The macOS build is still failing

**Comment by krishvoor** [comment]
Can you kindly trigger the job, please?

**Review by yurishkuro — APPROVED** [review]
Thanks!

**Pull request merged**